misc/python/materialize/cli/run.py

```diff
@@ -244,6 +241,8 @@ def main() -> int:
                     if p.name != "prometheus" and p.name != "tempo"
                 )
                 paths.extend(p for p in scratch.glob("*"))
+                tmp = pathlib.Path(tempfile.gettempdir())
+                paths.extend(p for p in tmp.glob("environmentd-*"))
```

Comment:
Highlighting this new cleanup inside the `/tmp` directory.

Reply:
This seems like it's only necessary because the RustRover configuration hardcodes a specific environment ID. That's mayyybe a fine approach but it's not really a supported configuration, so would rather not have run.py have special reset logic to account for that—too much action at a distance. Seems like something that should go in the RustRover-specific reset script instead.

Reply:
@benesch while I've reverted the changes, I do think this extra cleanup in `/tmp` should be part of the script. `--reset` does a full cleanup except for these files in `/tmp` and instead relies on unique environment ids. Across multiple `bin/environmentd --reset` runs, `/tmp` will continue to accrue multiple folders. Not that big a deal, but I think it would still be nice to explicitly mark all the places materialize stores data that might need a cleanup.

Happy to open another PR if you agree.